setup.py

```diff
@@ -30,7 +30,7 @@ def main():
 
     setuptools.setup(
         name="google-cloud-ndb",
-        version="0.0.1",
+        version="0.1.0",
```

Comment:
```suggestion
        version="0.0.1",
```
Please undo this change. the releasetool will use it to try and determine what release to make next.